assets/js/components/bapvCalendar.js

```diff
@@ -15,13 +15,18 @@ function handleSelectDate(event) {
 
   // highlight selected day
   document.querySelector(`.${calendarDayClass}[data-date='${dateToShow}']`).classList.add(calendarDaySelectedClass)
+  document.getElementById(`day-link-${dateToShow}`).blur()
```

Comment:
Workaround for an issue where a calendar grid date can remain focus (and therefore styled incorrectly) if the visit session `input` that is jumped to with `focus()` below is `disabled` - because `disabled` elements can't receive focus.

Will be looking for a better solution but this sorts it for now...